src/VideoCaptureWrap.cc

```diff
@@ -33,6 +33,7 @@ void VideoCaptureWrap::Init(Local<Object> target) {
   Nan::SetPrototypeMethod(ctor, "setHeight", SetHeight);
   Nan::SetPrototypeMethod(ctor, "setPosition", SetPosition);
   Nan::SetPrototypeMethod(ctor, "getFrameAt", GetFrameAt);
+  Nan::SetPrototypeMethod(ctor, "nFrames", NumFrames);
```

Comment:
How about making this method name more closely match the underlying CV property.  I suggest "getFrameCount".